app/HTML.js

```diff
@@ -29,10 +30,17 @@ const renderers = {
         }}
       />);
     }
+    else if (htmlAttribs.class === "wp-block-embed__wrapper") {
+      let uri = children[0][0].props.source.uri;
```

Comment:
Rather than doing this (it's not best practice to directly access children props if you can avoid it), could you try using something else like `passProps`?